apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -123,6 +123,11 @@ type OpenTelemetryTargetAllocator struct {
 	// The current options are least-weighted and consistent-hashing. The default option is least-weighted
 	// +optional
 	AllocationStrategy string `json:"allocationStrategy,omitempty"`
+	// FilterStrategy determines how to filter targets before allocating them among the collectors.
+	// The current options are no-op (no filtering) and relabel-config (drops targets based on prom relabel_config)
+	// The default is no-op
+	// +optional
+	FilterStrategy string `json:"filterStrategy,omitempty"`
```

Comment:
What would is the use case for `no-op`?  Distributing load more to the collector?     
Currently I'm tending to make `relabel-config` the normal behavior.

Reply:
Maybe @jaronoff97 has thoughts on this. One use case I can think of is for testing - maybe in some instances its important to know all the targets discovered before filtering. Also as you mentioned for load testing where increasing the targets would be helpful.